src/i18n.ts

```diff
@@ -84,7 +85,7 @@ export function t(path: string, replacement?: { [key: string]: string }) {
       translation = translation.replace(`{{${key}}}`, replacement[key]);
     }
   }
-  return translation;
+  return getShortcutKey(translation);
```

Comment:
Actually, this needs to be reverted and implemented some other way. Doing it this way would replace things it should (such as matches within english words etc.).